### ISSUE TITLE
Questions about the effect of pretraining.

### ISSUE THREAD

**Issue by NorbertZheng** [bbaaii/DreamDiffusion#13]
I'm not sure about the effect of EEG pretraining. I have directly train a shallow convolution network on `eeg_5_95_std.pth` as used in your paper. And the test accuracy according to max validation is about 60% on average, which is better than the classification accuracy 45% mentioned in your paper.

And a parallel work [NeuroImagen](https://arxiv.org/pdf/2308.02510.pdf) achieves better results without pretraining. So I wonder whether EEG pretraining is indeed meaningful???

**Comment by DILIU1** [comment]
好兄弟，可以加一个微信吗？我也在做这方面的工作，希望多多交流。

**Comment by zihang-xu** [comment]
I guess NeuroImagen may not use the same data, because the paper did not mention 5-95 hz filtering

**Comment by RomGai** [comment]
> I guess NeuroImagen may not use the same data, because the paper did not mention 5-95 hz filtering

I think that's true. Without 5-95 hz filtering, this dataset is unreliable.

**Comment by TwoHuang** [comment]
> 好兄弟，可以加一个微信吗？我也在做这方面的工作，希望多多交流。

好兄弟,可以加个微信吗,我也开始做这方面工作了

**Comment by feroooooo** [comment]
> > 好兄弟，可以加一个微信吗？我也在做这方面的工作，希望多多交流。
> 
> 好兄弟,可以加个微信吗,我也开始做这方面工作了

我也，请问还在做吗
